dev-util/perf/perf-6.6.ebuild

```diff
@@ -223,11 +223,29 @@ perf_make() {
 		disable_libdw=1
 	fi
 
+	# perf directly invokes LD for linking without going through CC, on mips
+	# it is required to specify the emulation.  port of below buildroot patch
+	# https://patchwork.ozlabs.org/project/buildroot/patch/20170217105905.32151-1-Vincent.Riera@imgtec.com/
+	local linker="$(tc-getLD)"
+	if use mips
+	then
+		if use big-endian
```

Comment:
Interestingly, none of these are set in profiles. See also https://bugs.gentoo.org/914881.

Anyway, I guess this is good enough for now, but we should see about including fixing this in the 23.0 rework for mips.